pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+markers =
+  sphinx
+
+filterwarnings =
+  ignore:'U' mode is deprecated:DeprecationWarning:
+  ignore:sphinx.builders.html.DirectoryHTMLBuilder is now deprecated.*:DeprecationWarning:
```

Comment:
I didn't realize this was deprecated and I can't seem to find details on it. When did this get deprecated? I don't see anything in the [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/builders/index.html).

Reply:
I have taken this file directly from sphinx-notfound-page -- https://github.com/rtfd/sphinx-notfound-page/blob/master/pytest.ini